src/loader/SpineResource.ts

```diff
@@ -1,28 +1,42 @@
-import { SkeletonData } from "@esotericsoftware/spine-core";
+import { AnimationStateData, Skeleton, SkeletonData } from "@esotericsoftware/spine-core";
 import { Engine, ReferResource, Texture2D } from "@galacean/engine";
-import { SpineAnimationRenderer } from "../SpineAnimationRenderer";
 
-export class SkeletonDataResource extends ReferResource {
-  readonly textures: Texture2D[] = [];
+export class SpineResource extends ReferResource {
```

Comment:
Try add instantiate method to improve useless!

Reply:
done.